.github/workflows/ci.yaml

```diff
@@ -20,16 +20,21 @@ jobs:
     steps:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
-      - name: Mount bazel action cache
+      # Cache build and external artifacts so that the next ci build is incremental.
+      # Because caches cannot be updated, create a new cache based on the contents of
+      # hashFiles (modify this to capture sources in your repo that would cause bazel
```

Comment:
I think you could add another step to the root README.md, pointing the user here.

Reply:
Added